packages/example-studio/components/FunkyEditor/FunkyEditor.js

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import SlateEditor from '@sanity/form-builder/lib/inputs/BlockEditor-slate'
+export default class FunkyEditor extends React.Component {
+  static propTypes = {
+    type: PropTypes.shape({
+      title: PropTypes.string
+    }).isRequired,
+    level: PropTypes.number,
+    value: PropTypes.array,
+    onChange: PropTypes.func.isRequired
+  };
+
+  handleChange = event => {
+    this.props.onChange(event)
```

Comment:
If all `handleChange` do is just call the given `onChange`-prop, maybe just pass it from render()?
```js
render() {
  const {type, value, level, onChange} = this.props
  return (
    //...
    <SlateEditor
      //...
      onChange={onChange}
     />
    //...
  )
}
```

Reply:
I think it has some value to show that you can make your own handler and do stuff. As you say in the below comment (readable.io).

Reply:
True, but then maybe we should add an example where we are actually doing something with the value (or at least a comment in `handleChange`describing that we _could_ do something with the value if we wanted to)